File: app/src/test/java/com/farmerbb/taskbar/mockito/BooleanAnswer.java
package com.farmerbb.taskbar.mockito;

import org.mockito.invocation.InvocationOnMock;
import org.mockito.stubbing.Answer;

public class BooleanAnswer implements Answer<Boolean> {
    public boolean answer = false;

    @Override
    public Boolean answer(InvocationOnMock invocation) {
        return answer;
    }
}

File: app/src/test/java/com/farmerbb/taskbar/mockito/BooleanAnswer.kt
package com.farmerbb.taskbar.mockito

import org.mockito.invocation.InvocationOnMock
import org.mockito.stubbing.Answer

class BooleanAnswer : Answer<Boolean> {
    var answer = false
    override fun answer(invocation: InvocationOnMock): Boolean {
        return answer
    }
}
